### ISSUE TITLE
The script printed every line of the file it ingested along with the parsing results

### ISSUE THREAD

**Issue by manish-drake** [gbmav/sql2puml#1]
Thanks for a nice contribution on the subject. I was looking for something quick to get a conversion done. I don't know any AWK. But I had to make some changes to the script to make it work for me.
I commented the following.
\# output_format="plantuml"
\# running=0

and I put the following inside braces.
{
    uml_parse_line($0)
}

**Comment by varontron** [comment]
I experienced this issue as well, and these fixes worked for me.  Thx.

**Comment by ufi68** [comment]
Yes tried me some hours before i found the solution form @manish-drake - thx a lot 👍 

I go also the Error:  
`fatal: Unmatched ( or \(: /(/`

I use awk/gawk 4.1.n 

I change the `("\(` and `("\)`  by `("\\(` and `("\\)` and it works fine now for me 

Thx for the tool we can now document our database structure inside our pipeline 💯
